ElementX/Sources/Screens/CreateRoom/View/CreateRoomScreen.swift

```diff
@@ -150,22 +156,73 @@ struct CreateRoomScreen: View {
                                     iconAlignment: .top),
                     kind: .selection(isSelected: !context.isRoomPrivate) { context.isRoomPrivate = false })
         } header: {
-            Text(L10n.commonSecurity.uppercased())
+            Text(L10n.screenCreateRoomRoomVisibilitySectionTitle.uppercased())
```

Comment:
Not related to your changes, but I don't think we need the `uppercased()` here, it'll be handled by the header automatically.
```suggestion
            Text(L10n.screenCreateRoomRoomVisibilitySectionTitle)
```